front/src/modules/events/hooks/useTrackPageView.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
```

Comment:
I feel the module should be called telemetry. Events seems more broad and we are dealing with telemetry here

Reply:
I'm not sure it's going to be just for external Telemetry. I feel like events could be used within the app (e.g. Thomas was talking about displaying usage statistics on commandbar, we could display usage context on pages also like who has view this note, etc.). We could eventually decide it's better to separate requests/packages but I feel like there's no wrong in letting it share a common structure instead of sending frontend events twice

Reply:
Yes Analytics is better! Server side too then?

Reply:
I guess it's better if the name is the same on both side :)